site-src/geps/gep-713.md

```diff
@@ -1,25 +1,35 @@
-# GEP-713: Policy Attachment
+# GEP-713: Metaresources and Policy Attachment
 
 * Issue: [#713](https://github.com/kubernetes-sigs/gateway-api/issues/713)
 * Status: Experimental
 
 ## TLDR
 
-This GEP aims to standardize policy attachment to resources associated with
-Gateway API by establishing a pattern which defines how `Policy` API types can
-have their relevant effects applied to network traffic. Individual policy APIs
-(e.g. `TimeoutPolicy`, `RetryPolicy`, etc) will include a common `TargetRef`
-field in their specification to identify how and where to apply that policy.
+This GEP aims to standardize attachment of one Kubernetes object to another in
```

Comment:
I think this may be too vague. Probably the most used forms of attachment in Gateway API will be Route -> Parent and Route -> Backend, neither of which use the models described in this doc.

Reply:
I've updated here, PTAL.